websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Cards
+---
+
+ * The actual card content is markdown
+ * Title have no linkable / hover anchor like section headings
+ * Titles never become entries in the page index nav like section headings
+ * Either an icon or an image path can be set, but not both.
+   * To start with, Images are _not_ supported for free-form in-content cards, only in places that are fully designed and implemented by developers. An Image is sized into the width of the card.
+   * Icons are supported anywhere but only those that are officially provided by the kit. An icon is sized into a fixed 48x48px square.
+ * Cards can occur in two interaction variants, "active" (visually elevated) and "flat" (visually an outline only).
+ * Cards must be embedded into a wrapper component that defines the responsive sizes (two aside / three aside) and parameters that must be identical in a give row or grid (active card area, the heading size)
+
+# "Cards" Wrapper
+
+ Properties (for authors):
+ * `active` (boolean): if set, the whole area of all child cards is a link, the card is elevated and has a hover effect if a link url is set.
+ * `narrow` (boolean): by default, two cards are shown side by side (except mobile viewports). if `narrow` is passed, the minimum card width is reduced so that three cards fit the content width on microsite landing pages.
```

Comment:
The cards get narrower in all views except the mobile one, hence the idea to call it "narrow".   
The component is called "cards" so it's verbally "narrow cards". 

I tried to avoid explicitly referencing the number of columns because that would become a content maintenance nightmare if we change something to the responsive behaviour or layout that leads to e.g. allowing four aside in some places.  

I'll ask the docs team, they are the actual users.

Reply:
No intense opinions there.